h/templates/panels/back_link.html.jinja2

```diff
@@ -0,0 +1,4 @@
+{% if back_location and back_label %}
+<a class="link link--footer"
```

Comment:
Having `link--footer` in here seems a bit odd if this is supposed to be a "reusable" panel...?